src/mixins/_button.scss

```diff
@@ -57,6 +64,7 @@
   border-style: solid;
   border-width: size.$edge-control;
   color: color.$text-light-emphasis;
+  color: var(--theme-color-text-button-default);
```

Comment:
I don't know if it's necessary anymore to have these non-custom-property fallbacks present, but I didn't feel confident enough in that to strip this out or dramatically change the way we are applying styles.

Reply:
For sure — I think you're right that we could remove most/all of these fallbacks but I agree that can happen later